### PR TITLE
Remove references to the IMPES algorithm used earlier.

### LINKED ISSUE #925: Do not use the word IMPES
This is currently used in the code, the parameters, and the manual. It turns out this is an incorrect term to use (it was only reasonable when we used to handle advection explicitly).

### PR DESCRIPTION
**Pull request by bangerth**
Instead mention it in a historical context.

Fixes #925.

### CODE REVIEW

**Comment by tjhei** [comment]
This is good, but it doesn't fix #925, because we still use it as the name for the (non)linear solver, see #972.

**Comment by bangerth** [comment]
Ah, I see. May I punt and leave that for a later pull request?

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -1508,9 +1508,17 @@ \subsubsection{Almost linear models}
 instead in each time step solves first the Stokes system with the previous
 time step's temperature, and then uses the so-computed velocity to solve the
 temperature equation. These kind of time stepping schemes are often referred
-to as \textit{IMPES} methods (they originate in the porous media flow
+to as \textit{operator splitting} methods. A particular operator
+splitting method, used in
+earlier \aspect{} versions, solves first the Stokes equations and then
+uses an explicit time stepping method for the temperature equation;
```
Comment (tjhei) [inline doc/manual/manual.tex]: Only the convective term is explicit, so this is not "an explicit time stepping method". Can you please fix this?

**Comment by tjhei** [comment]
Can you address the comment, merge, and then re-open #925?

**Comment by tjhei** [comment]
> Ah, I see. May I punt and leave that for a later pull request?

sure. Please address the other comment, merge, and then re-open #925.

**Pull request merged**